Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM progrium/busybox
+FROM golang:1.5.3-alpine
 
-RUN opkg-install ca-certificates
+ENV CONCOURSE_CODE_PATH ${GOPATH}/src/github.com/concourse/s3-resource
 
-# satisfy go crypto/x509
-RUN for cert in `ls -1 /etc/ssl/certs/*.crt | grep -v /etc/ssl/certs/ca-certificates.crt`; \
-      do cat "$cert" >> /etc/ssl/certs/ca-certificates.crt; \
-    done
+RUN apk add --update git bash \
+  && rm -rf /var/cache/apk/*
 
-ADD built-check /opt/resource/check
-ADD built-in /opt/resource/in
-ADD built-out /opt/resource/out
+ADD . /code
```

Comment:
Is there any reason that it's not `ADD . ${GOPATH}/src/github.com/concourse/s3-resource` ?